packages/venia-ui/lib/components/CheckoutPage/ShippingMethod/shippingMethod.js

```diff
@@ -0,0 +1,30 @@
+import React, { useCallback } from 'react';
+
+import Button from '../../Button';
+
+import { renderIf } from '../utilities';
+
+import defaultClasses from './shippingMethod.css';
+
+export default ({ onSave, doneEditing }) => {
```

Comment:
This'll get changed when someone implements this but we don't destructure props in the signature for function components anywhere else in this repo.

Reply:
Yeah, I was aware of that. Missed to change this. Should have changed it. Will do 👍